docs/multi-agent-manager.md

```diff
@@ -22,59 +26,93 @@ We first add a couple of finished AEA project:
 ``` python
 from aea.configurations.base import PublicId
 
-weather_client_id = PublicId.from_str("fetchai/weather_client:0.21.0")
 weather_station_id = PublicId.from_str("fetchai/weather_station:0.20.0")
-manager.add_project(weather_client_id)
+weather_client_id = PublicId.from_str("fetchai/weather_client:0.21.0")
 manager.add_project(weather_station_id)
+manager.add_project(weather_client_id)
+weather_station_name = weather_station_id.name
+weather_client_name = weather_client_id.name
 ```
 
 ## Adding agent instances
 
 Save the following private keys in the respective files.
 ``` python
-FET_PRIVATE_KEY_1 = b"72d3149f5689f0749eaec5ebf6dba5deeb1e89b93ae1c58c71fd43dfaa231e87"
-FET_PRIVATE_KEY_PATH_1 = Path("fetchai_private_key_1.txt")
-FET_PRIVATE_KEY_PATH_1.write_bytes(FET_PRIVATE_KEY_1)
+FET_PRIVATE_KEY_STATION = b"72d3149f5689f0749eaec5ebf6dba5deeb1e89b93ae1c58c71fd43dfaa231e87"
+FET_PRIVATE_KEY_PATH_STATION = Path(KEYS_DIR, weather_station_name, "fetchai_private_key.txt").absolute()
+os.makedirs(Path(KEYS_DIR, weather_station_name))
+FET_PRIVATE_KEY_PATH_STATION.write_bytes(FET_PRIVATE_KEY_STATION)
+
+FET_CONNECTION_PRIVATE_KEY_STATION = b"bf529acb2546e13615ef6004c48e393f0638a5dc0c4979631a9a4bc554079f6f"
+FET_CONNECTION_PRIVATE_KEY_PATH_STATION = Path(KEYS_DIR, weather_station_name, "fetchai_connection_private_key.txt").absolute()
+FET_CONNECTION_PRIVATE_KEY_PATH_STATION.write_bytes(FET_CONNECTION_PRIVATE_KEY_STATION)
+
+CERT_PATH_STATION = Path(CERTS_DIR, weather_station_name, "conn_cert.txt").absolute()
 
-FET_CONNECTION_PRIVATE_KEY_1 = b"bf529acb2546e13615ef6004c48e393f0638a5dc0c4979631a9a4bc554079f6f"
-FET_CONNECTION_PRIVATE_KEY_PATH_1 = Path("fetchai_connection_private_key_1.txt")
-FET_CONNECTION_PRIVATE_KEY_PATH_1.write_bytes(FET_CONNECTION_PRIVATE_KEY_1)
+SOEF_STORAGE_PATH_STATION = Path(KEYS_DIR, f"{weather_station_name}_soef_token.txt").absolute()
+
+BUILD_PATH_STATION = Path(WORKING_DIR, weather_station_id.author, weather_station_name, ".build", "connection", "fetchai", "p2p_libp2p").absolute()
```

Comment:
This is suboptimal. The problem here is that the connections/modules in general are not aware of the relevant aea project directory. So they create everything relative to root.